oscarapi/app.py

```diff
@@ -0,0 +1,20 @@
+from oscar.core.application import Application
+from oscar.views.decorators import permissions_required
+
+from oscarapi.urls import urlpatterns
+
+
+class RESTApiApplication(Application):
+    def get_url_decorator(self, pattern):
+        "Fixes bug in Application.get_url_decorator wich would decorate None values."
```

Comment:
I ported your changes in https://github.com/django-oscar/django-oscar/commit/8a6ac445579a40f1dcb100fd047c88760752a199. Thanks. What was the issue it caused?